src/Bridge.cpp

```diff
@@ -213,11 +237,72 @@ std::string Bridge::requestUsername()
     return username;
 }
 
+bool Bridge::StartStreaming(std::string group_identifier)
+{
+    nlohmann::json request;
+
+    request["stream"]["active"] = true;
+
+    nlohmann::json answer;
+
+    std::string uri = "/api/" + username + "/groups/" + group_identifier;
+
+    answer = http_handler->PUTJson(uri, request, ip, port);
+
+    if(answer[0].contains("success"))
```

Comment:
This will fail if answer is empty. It is safer to use:
```suggestion
    nlohmann::json success = utils::safeGetMember(answer, 0, "success", key);
    return success == true;
```